submissions/m-ruslan/html-css-popup/index.html

```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="app.css" />
+    <link rel="icon" type="image/png" href="favicon.png" />
+    <title>Popup</title>
+  </head>
+  <body>
+    <nav>
+      <ul class="nav-container">
+        <li class="nav-el">
+          <a href="#" tabindex="0">Mail</a>
+        </li>
+        <li class="nav-el">
+          <a href="#" tabindex="0">Images</a>
+        </li>
+
+        <li class="nav-el">
+          <input type="checkbox" id="button-1"  tabindex="0">
+          <a href="#" tabindex="-1">
```

Comment:
I guess native link tag (`a`) should direct user somewhere, but in this case better to use
`button` and simple `div` tags.

Reply:
Thank you, Artem.
I've removed this `a` tag because it caused an error in html validator.